source/services/time/time.service.tests.ts

```diff
@@ -1,16 +1,36 @@
 import { ITimeUtility, TimeUtility } from './time.service';
 import { CompareResult } from '../../types/compareResult';
+import { stringUtility } from '../string/string.service';
 
 describe('timeUtility', () => {
 	let timeUtility: ITimeUtility;
 
 	beforeEach(() => {
-		timeUtility = new TimeUtility();
+		timeUtility = new TimeUtility(stringUtility);
 	});
 
 	it('should compare times and return a compare result to indicate which is greater', (): void => {
 		expect(timeUtility.compareTimes('12:00PM', '1:00PM')).to.equal(CompareResult.less);
 		expect(timeUtility.compareTimes('12:00PM', '12:00PM')).to.equal(CompareResult.equal);
 		expect(timeUtility.compareTimes('2:00PM', '1:00PM')).to.equal(CompareResult.greater);
 	});
+
+	it('should return null if the time is null', (): void => {
+		expect(timeUtility.parseTime(null)).to.be.null;
+		expect(timeUtility.formatTime(null)).to.be.null;
+	});
+
+	it('should parse the time string into a time object', (): void => {
+		expect(timeUtility.parseTime('8:00AM')).to.deep.equal({ hour: 8, minute: 0, period: 'AM' });
+		expect(timeUtility.parseTime('12:00PM')).to.deep.equal({ hour: 12, minute: 0, period: 'PM' });
```

Comment:
Good idea. For now we're just sticking with AM/PM time, unless someone pushes for 24 hour time and we agree to it. This is largely due to the complexity of switching everything over, such as dateTime and any components still in RL2